deployments/hadoop-yarn/bin/deploy.sh

```diff
@@ -19,6 +19,21 @@
 # </meta:header>
 #
 
+# ----------------------------------------------------------------
+# Check if we are deleting live, confirm before continuing if yes
+
+
+live_hostname=$(ssh  -o "StrictHostKeyChecking no" fedora@live.gaia-dmp.uk 'hostname')
+
+if [[ "$live_hostname" == *"$cloudname"* ]]; then
+    read -p "You are deleting the current live system!! Do you want to proceed? (y/n) " -n 1 -r
```

Comment:
Small syntax changes:
1. this is in create-all, so it won't just delete the live system, it will replace it 
2. the command line convention is to capitalize the default setting
 
```suggestion
    read -p "You are replacing the current live system!! Do you want to proceed? (y/N) " -n 1 -r
```

Reply:
Having 2 checks, once in `create-all` and then again in `delete-all` is not a bad thing. Deleting and replacing the live service like this is something we shouldn't be doing at this stage. So asking for confirmation twice is a good thing.